src/workflows/workflows_test.ts

```diff
@@ -173,3 +173,37 @@ Deno.test("Workflows properly treats interactivity and user context types", () =
     `{{steps.0.user.secret}}`,
   );
 });
+
```

Comment:
I don't know if it's common to document _why_ certain tests exist, but in this case, I can see the value in a comment that explains what I wrote [here](https://github.com/slackapi/deno-slack-sdk/pull/123#discussion_r1046372790).

Something like:

Deno will exit on uncaught exceptions. JSON.parse will raise an exception when given `undefined`. We can set `undefined` manually here in the case of an `undefined` inputValue, which will allow Deno to continue and pass the undefined values up to the validation API -- which will communicate back to the CLI the specific validation errors it ran in to. 

Or something like that.

Reply:
That sounds good to me!

I really like this approach to documenting/explaining test cases as it keeps the knowledge within the codebase but doesn't introduce any bloat or distraction into the codepath. I think we should do this elsewhere as well, maybe as we do our exercise @filmaj ?

Reply:
100% useful, I try to do that as much as possible myself (esp. via test titles, e.g. https://github.com/slackapi/deno-slack-sdk/compare/main...param-tests-custom-types-moarprops#diff-9dc41c90f103a2563c551a62f729231125264b51509240a83e38117f4e50feb7R62)